test/test_helper.rb

```diff
@@ -1,5 +1,7 @@
 root_location = File.expand_path("../../", __FILE__)
 inc_dirs = Dir.glob("#{root_location}/library/*/src")
+# Needed to test Y2DIR support in Yast::Directory
+inc_dirs << "#{root_location}/library/general/test"
```

Comment:
to be honest I do not like it...better add support to respect previous Y2DIR option like 
```
  if ENV["Y2DIR"]
    ENV["Y2DIR"] << ":"
  else
    ENV["Y2DIR"] = ""
  end
  ENV["Y2DIR"] << inc_dirs.join(":")
```

and set Y2DIR in respective test, because if more tests need own testing location if can quite quickly become horrible mess